MFE/dashboard/src/i18n/translations/en.success.i18n.json

```diff
@@ -24,7 +24,7 @@
     "deleteMultipleCertificates": "All certificates were deleted",
     "disassociateDevice": "Certificate disassociated from device",
     "associateDevice": "Certificate associated with the device",
-    "createCertificate": "Certificate created",
```

Comment:
Here, I have removed this line because there is already a translation for 'createCertificate' on line 22 of this file